docs/xconcurrentbackground.md

```diff
@@ -30,9 +30,11 @@ Specifies the number of low-priority background threads attached to assist the m
 
         -Xconcurrentbackground<n>
 
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** This value is reported in the initialized header of Verbose GC logs as "gcthreads Concurrent Mark" attribute.
```

Comment:
@RSalman - I was a little confused by the note until I looked at a log. Perhaps you can add the actual output. Suggest:

This value is reported in the header section of a verbose GC log with an entry `<attribute name="gcthreads Concurrent Mark" value="1" />`.

Reply:
Thanks - updated with the suggested text